code/game/machinery/bots/cprbot.dm

```diff
@@ -0,0 +1,202 @@
+/obj/structure/machinery/bot/cprbot
+	name = "CPRbot"
+	desc = "Designed for urgent medical intervention, this CPRbot offers high-tech support in a compact form."
+	icon = 'icons/obj/structures/machinery/aibots.dmi'
+	icon_state = "cprbot0"
+	density = FALSE
+	anchored = FALSE
+	health = 100
+	maxhealth = 100
+	req_access = list(ACCESS_MARINE_MEDBAY)
+	var/const/search_radius = 10
```

Comment:
That range is way to high. Not for balance, but for sanity. Screen size is 7 tile radius. 10 means it goes to stuff you cant even see. At a very fast speed.

Reply:
Alright it's an easy fix really I can tone it down to 7 so it never leaves the players screen at least.